ops/systemd/export_workers.env

```diff
@@ -1,16 +1,23 @@
 ## General settings
-HOSTNAME=https://export.hotosm.org
+HOSTNAME=export-staging.hotosm.org
 DJANGO_SETTINGS_MODULE=core.settings.project
 
 ## SMTP Settings
-EMAIL_HOST_PASSWORD=<SECURE_SMTP_PASSWORD>
 EMAIL_HOST=smtp.mailgun.org
-EMAIL_HOST_USER=export@examplemail.com
+EMAIL_HOST_USER=export@example.org
+EMAIL_HOST_PASSWORD=<password>
+REPLY_TO_EMAIL=sysadmin@example.org
+
+##OAUTH 
+OSM_API_KEY=<Secret Key>
+OSM_API_SECRET=<OSM API Secret>
+
+## Workers
+WORKER_SECRET_KEY=<Generate Secret Key for workers>
 
 ## Storage options
-EXPORT_DOWNLOAD_ROOT=/mnt/data/downloads
-EXPORT_STAGING_ROOT=/mnt/data/staging
-PLANET_FILE=/mnt/data/planet/planet.osm.pbf
+EXPORT_DOWNLOAD_ROOT=/data/downloads
+EXPORT_STAGING_ROOT=/data/staging
```

Comment:
The examples correspond to the strings mentioned in the documentation.

Can be the same or a replaceable template key

```
EXPORT_DOWNLOAD_ROOT=<DOWNLOAD_ROOT>
EXPORT_STAGING_ROOT=<STAGING_ROOT>
```

Ideally, I would like the keys to have a different string. I will create an issue for this.

Reply:
okay